sandbox/grommet-app/src/pages/layouts/kinds/RecordDetail/ILOSecurity.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Box, NameValueList, NameValuePair } from "grommet";
+import { sentenceCase } from "../../../../utils/format";
+import { statusMap } from "../../../../utils/status";
+
+const iLOSecurity = {
+  'critical': {
+    servers: [],
+    label: 'At risk',
+  },
+  'warning': {
+    servers: [{ id: 'server1', name: 'Server 1', url: '#' }],
+    label: 'Needs attention',
+  },
+  'ok': {
+    servers: [{ id: 'server2', name: 'Server 2', url: '#' }],
+    label: 'Okay',
+  },
+  'unknown': {
+    servers: [],
+    label: 'Unknown',
+  },
+  'unsupported': {
+    servers: [],
+    label: 'Unsupported',
+  },
+}
+
+const nameProps = {
+  width: ['xsmall', 'max-content']
```

Comment:
Should we include this in grommet-theme-hpe v6.0.0 as the default?

Reply:
oooh I like your thinking.

Perhaps we could do something similar for bottom pad on PageHeader. Maybe audit some other "we always seem to apply these props".